docs/reference/search/request/sort.asciidoc

```diff
@@ -337,8 +337,7 @@ Allow to sort based on custom scripts, here is an example:
 }
 --------------------------------------------------
 
-Note, it is recommended, for single custom based script based sorting,
-to use `function_score` query instead as sorting based on score is faster.
+Note, this has the same effect as a `function_score` query with a single script function and '"boost_mode": "replace"'.
```

Comment:
I'm not sure it helps more than it adds confusion, maybe we should just drop it entirely?